A/ADOLC/common.jl

```diff
@@ -0,0 +1,420 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+#include("../../fancy_toys.jl") # for get_addable_spec
```

Comment:
Why are you duplicating libjulia's `common.jl` as well as `fancy_toys.jl` in here? That seems rather problematic to me.

Everything else I commented is sorta optional, but this one seems serious to me.

Reply:
Ah, this appears to be a leftover from trying locally.